swagger_spec_validator/validator20.py

```diff
@@ -21,5 +22,101 @@ def validate_spec_url(spec_url):
 
 
 def validate_spec(spec_json):
+    """Validates a Swagger 2.0 API Specification given a json dict.
```

Comment:
I think the use of "json dict" can be confusing here. It's true that the dict was created by parsing json, but after that point, the fact that it was json isn't really relevant. json was just the serialization format.

Could we use a name that conveys more meaning about the contents?

If this is a swagger spec, maybe `swagger_spec` ?